app/models/aws_s3.rb

```diff
@@ -148,11 +148,11 @@ def put(file_name:, prefix:)
 
   def store(content:, name:, content_type: nil)
     obj = @bucket.object(name)
-    if Rails.env.development? || Rails.env.test?
-      obj.put(body: content, content_type: content_type)
-    else
-      obj.put(body: content, server_side_encryption: 'AES256', content_type: content_type)
-    end
+    args = { body: content }
+    args.merge!(content_type: content_type) if content_type
+    # TODO: document why we skip encryption here
```

Comment:
So, I believe we're skipping server side encryption for test and development because minio didn't (or doesn't) support it, or it was fairly complex to setup and didn't seem necessary for local development or testing.  Might be better to inspect the hostname, and ensure it's enabled for real s3 urls.